code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -251,6 +251,54 @@
 			qdel(R)
 	T.Bless()
 
+/datum/reagent/hydrogen_peroxide
+	name = "Hydrogen peroxide"
+	description = "An ubiquitous chemical substance that is composed of hydrogen and oxygen and oxygen."
+	color = "#AAAAAA77" // rgb: 170, 170, 170, 77 (alpha)
+	taste_description = "burning water"
+	var/cooling_temperature = 2
+	glass_icon_state = "glass_clear"
+	glass_name = "glass of oxygenated water"
+	glass_desc = "The father of all refreshments. surely it tastes great? right.."
```

Comment:
Grammatical error. "The father of all refreshments. Surely it tastes great, right?" or "The father of all refreshments. Surely it tastes great? Right..." -- note ellipsis (...) have three periods if you decide to keep it.